pkg/task/cluster.go

```diff
@@ -93,13 +114,13 @@ func retrieveClusterFromEnvironment() *model.ClusterDetails {
 		cluster.Name = nameFromEnv
 	}
 	if urlFromEnv := os.Getenv("CLUSTER_URL"); urlFromEnv != "" {
-		cluster.URL = urlFromEnv
+		cluster.Connection.Server = urlFromEnv
```

Comment:
`cluster.Connection` will be nil here I think. I'm not actually sure if this code path is used anywhere, though. We might be able to remove this method entirely?

Reply:
Yeah sure, if this isn't used then I'd be happy to remove it. I don't see any occurrences of these env vars in our github org.